cadc-inventory-db/src/main/resources/inventory.Artifact.sql

```diff
@@ -9,7 +9,8 @@ create table <schema>.Artifact (
     contentEncoding varchar(128),
 
     siteLocations uuid[],
-    storageLocation varchar(256),
+    storageLocation_storageID varchar(256),
+    storageLocation_storageBucket varchar(8);
```

Comment:
Do you think 8 is long enough?  Perhaps it should be 32 to support UUIDs?

Reply:
For the bucket you want plain strings so you can support a shorter prefix (smaller number of larger buckets). 5 chars of hex is 1 million buckets. AD would use the archie name here, which fits into 8.

Reply:
Of course, we can trivially make it longer later if needed.

Reply:
Presumably the storageBucket is for non-standard use (like with ad) so doesn't necessarily support bucket prefixes.  But yes, we can always extend it.